toolkit/Makefile

```
@@ -12,14 +12,17 @@ toolkit_root := $(abspath $(dir $(lastword $(MAKEFILE_LIST))))
 SCRIPTS_DIR  ?= $(toolkit_root)/scripts
 
 # Build targets
+##help:var:CONFIG_FILE:<config_path>=Path to image configuration file to use.
 ifeq ($(origin CONFIG_FILE), undefined)
 CONFIG_FILE             = $(toolkit_root)/imageconfigs/core-efi.json
-$(warning CONFIG_FILE is undefined, defaulting to toolkit's core-efi.json.)
+#$(warning CONFIG_FILE is undefined, defaulting to toolkit's core-efi.json.)
 endif
 CONFIG_BASE_DIR         ?= $(dir $(CONFIG_FILE))
 PACKAGE_BUILD_LIST      ?=
+##help:var:PACKAGE_REBUILD_LIST:<spec_list>=List of spec basenames to force rebuild.
```

A small tweak, so people don't accidentally try to use `kernel.spec` as a valid argument + a mention how to input multiple specs:
```suggestion
##help:var:PACKAGE_REBUILD_LIST:<spec_list>=List of space-separated spec folders to force rebuild. Example: PACKAGE_REBUILD_LIST="kernel go which".
```